tutorials/classical_simulation/Simulating with SPAM errors.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>Proposition (if that is right) "As η grows, more <strong>atoms</strong> are badly-prepared (i.e, pumped into a state different from |g⟩) and <strong>less atoms are found in the register"</strong></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Actually, you will see more atoms because they are there, just in a state that does not respond to the laser sequence. So they are not expelled at the end because they never go into the Rydberg state.</p>